### PR TITLE
feat: arbitrum nova placeholder assets

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Coingecko only provides a handful of asset for arbitrum nova. Add placeholder assets if missing from coingecko to ensure the user can see their balance, tx history, and interact with the assets.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Low - additive and specific to arbitrum nova

## Testing

- Ensure assets that don't exist on coingecko are added and you can see the corresponding balance and tx history

### Engineering

:point_up: 

### Operations

:point_up: 

## Screenshots (if applicable)

image

### CODE REVIEW

File: src/state/slices/portfolioSlice/portfolioSlice.ts [hunk 1]
```diff
@@ -126,7 +129,38 @@ export const portfolioApi = createApi({
...
+                  },
+                  { byId: {}, ids: [] },
+                )
+
+                // upsert placeholder assets
+                dispatch(assetSlice.actions.upsertAssets(assets))
+
+                return accountToPortfolio({
+                  portfolioAccounts,
+                  assetIds: assetIds.concat(assets.ids),
```
Comment (kaladinlight) [inline src/state/slices/portfolioSlice/portfolioSlice.ts:155]: chose this as it seemed more efficient than awaiting the `upsertAssets` and using the state directly

File: packages/chain-adapters/src/evm/types.ts [hunk 2]
```diff
@@ -6,7 +6,13 @@ import type * as common from '../types'
...
   nonce: number
-  tokens?: common.AssetBalance[]
+  tokens?: Token[]
+}
+
+export type Token = common.AssetBalance & {
+  symbol: string
+  name: string
+  precision: number
 }
```
Comment (woodenfurniture) [inline packages/chain-adapters/src/evm/types.ts:16]: can use the type `MinimalAsset` from asset service

```suggestion
  tokens?: Token[]
}

export type Token = common.AssetBalance & MinimalAsset
```

Comment (woodenfurniture) [reply packages/chain-adapters/src/evm/types.ts:16]: would it worth commenting that the additional metadata is used for arb nova placeholder assets?

Comment (kaladinlight) [reply packages/chain-adapters/src/evm/types.ts:16]: I can add a quick comment regarding the metadata, but imo I think separation between chain-adapters and asset service makes sense and would rather not start mixing types there.

**Review by woodenfurniture — APPROVED** [review]

**Pull request merged**